### PR TITLE
[CA-1078] Working migration endpoint

### PR DESCRIPTION
**Pull request by s-rubenstein**
This adds a working endpoint to put a workspace in the queue to migrate. It intentionally does not handle 'getMigrationStatus', because that is both seperable, and would require some thinking about how we want to compose that response. Strictly speaking it also isn't necessary for our OKR, as we can just go into the DB to check it.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [ ] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell your tech lead (TL) that the PR exists if they want to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitor.scala [hunk 1]
```diff
@@ -95,6 +95,13 @@ object V1WorkspaceMigrationMonitor
 
   import slick.jdbc.MySQLProfile.api._
 
+  final def isInQueueToMigrate(workspace: Workspace): ReadAction[Boolean] =
```
Comment (s-rubenstein) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitor.scala:98]: This is used for testing, especially while we don't have anything that moves the queue along.

**Comment by s-rubenstein** [comment]
https://broadworkbench.atlassian.net/browse/CA-1645 captures the get endpoint.

**Comment by ehigham** [comment]
Looks good but I don't think we want to document this right now. Perhaps we wait until we're ready to go?

**Comment by s-rubenstein** [comment]
I can remove it from swagger. That sounds good.

File: core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 2]
```diff
@@ -2094,6 +2095,16 @@ class WorkspaceService(protected val userInfo: UserInfo,
     } yield hasAccess
   }
 
+  def migrateWorkspace(workspaceName: WorkspaceName): Future[Unit] = {
+    logger.info(s"migrateWorkspace - workspace:'${workspaceName.namespace}/${workspaceName.name}'")
```
Comment (andy7i) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala]:
```suggestion
    logger.info(s"migrateWorkspace - workspace:'${workspaceName.namespace}/${workspaceName.name}' is being scheduled for migration")
```

**Review by andy7i — APPROVED** [review]
1 suggestion ; 👍

**Review by ehigham — APPROVED** [review]
Looks great. Thanks.

**Pull request merged**